src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -471,7 +473,7 @@ public function getEditForm()
             );
     }
 
-    public function getAllocateForm()
+    public function getAllocateForm($num_weeks, $startdate)
```

Comment:
Would it not be better to do the `MyRadio_Scheduler::getActiveApplicationTermInfo()` call inside this function? Feels a bit weird for the form to take that in as a parameter. Feel free to disregard though.